src/web/lib/hakuneko/frontend@classic-dark/chapters.html

```diff
@@ -574,16 +581,16 @@
             }
 
             /**
-             * 
+             *
              */
             onChapterStatusChanged( e ) {
                 let chapter = e.detail;
                 if( !this.chapterList
                     || !this.selectedManga
                     || !this.selectedConnector
                     || this.selectedManga.id !== chapter.manga.id
-                    || (this.selectedConnector.id !== this.bookmarkConnectorID 
-                        && this.selectedConnector.id !== this.clipboardConnectorID 
+                    || (this.selectedConnector.id !== this.bookmarkConnectorID
+                        && this.selectedConnector.id !== this.clipboardConnectorID
```

Comment:
same story here, nothing changed nothing added...

Reply:
Your change is correct, trailing white-spaces must be removed according to the lint configuration.